src/components/grouped-character/grouped-character.test.tsx

```diff
@@ -0,0 +1,451 @@
+import React, { useState } from "react";
+import { render, screen } from "@testing-library/react";
+
+import userEvent from "@testing-library/user-event";
+import GroupedCharacter, { CustomEvent } from "./grouped-character.component";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+import FormFieldStyle from "../../__internal__/form-field/form-field.style";
+import Logger from "../../__internal__/utils/logger";
+
+jest.mock("../../__internal__/utils/logger");
+
+const MockComponent = ({
```

Comment:
**suggestion(non-blocking)**: what do you think about renaming this to `ControlledGroupedCharacter`, so its more descriptive? Not essential though.